config/routes.rb

```diff
@@ -3,7 +3,7 @@
   devise_for :users
 
   root controller: :images, action: :index
-
+  resources :checkout 
```

Comment:
I agree that this functionality should be in the ShoppingCartsController but I would actually propose having a show method first. That is the first thing you guys are doing right? Showing the content of the cart.